packages/cli/src/lib/cli/cliProcess.js

```diff
@@ -92,12 +92,14 @@ module.exports = class CLIProcess extends CLICommand {
     }
 
     /**
-     * Starts or stops a single adapter. If there are multiple instances this fails
+     * Starts or stops a single or all instances of adapter.
+     * If there are multiple instances this fails if flag allInstances is not set.
      * @param {string} adapter The adapter to start
      * @param {boolean} enabled
      * @param {boolean} [restartIfRunning=false] Whether running instances should be restarted
+     * @param {boolean} [allInstances=false] Whether all running instances should be restarted
      */
-    setAdapterEnabled(adapter, enabled, restartIfRunning) {
+    setAdapterEnabled(adapter, enabled, restartIfRunning, allInstances) {
```

Comment:
We should transition this to use an options object instead of appending parameters IMO, but this is something for a separate cleanup PR.